.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ["2.5", "2.6", "2.7", "3.0"]
+        ruby-version: ["3.2", "3.3", "3.4"]
```

Comment:
@andyw8 Considering the type of projects we encounter at [work](https://www.fastruby.io/monthly-ruby-maintenance), I'm not interested in dropping support for ancient rubies. There are still a ton of companies out there that still use rubies < 3.0

Reply:
Ok, I see, let me update.